modules/external-secrets-enable-operator-metrics.adoc

```diff
@@ -0,0 +1,202 @@
+    name: external-secrets-operator-controller-manager
+    namespace: external-secrets-operator
+----
+
+.. Create the `ClusterRoldeBinding` custom resource by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f clusterrolebinding-external-secrets.yaml
+----
+
+. Create the `ServiceMonitor` CR if using the default `HTTPS`:
+
+.. Create the `servicemonitor-external-secrets-operator-https.yaml` YAML file:
++
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: external-secrets-operator
+  name: external-secrets-operator-metrics-monitor
+  namespace: external-secrets-operator
+spec:
+  endpoints:
+    - authorization:
+        credentials:
+          name: external-secrets-operator-metrics-auth
+          key: token
+        type: Bearer
+      interval: 60s
+      path: /metrics
+      port: metrics-https
+      scheme: https
+      scrapeTimeout: 30s
+      tlsConfig:
+        ca:
+          configMap:
+            name: openshift-service-ca.crt
+            key: service-ca.crt
+        serverName: external-secrets-operator-controller-manager-metrics-service.external-secrets-operator.svc.cluster.local
+  namespaceSelector:
+    matchNames:
+      - external-secrets-operator
+  selector:
+    matchLabels:
+      app: external-secrets-operator
+      svc: external-secrets-operator-controller-manager-metrics-service
+----
+
+.. Create the `ServiceMonitor` CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f servicemonitor-external-secrets-operator-https.yaml
+----
+
+. Create the `ServiceMonitor` CR if configured to use `HTTP`:
+
+.. Create the `servicemonitor-external-secrets-operator-http.yaml` YAML file:
++
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: external-secrets-operator
+  name: external-secrets-operator-metrics-monitor
+  namespace: external-secrets-operator
+spec:
+  endpoints:
+    - authorization:
+        credentials:
+          name: external-secrets-operator-metrics-auth
+          key: token
+        type: Bearer
+      interval: 60s
+      path: /metrics
+      port: metrics-http
+      scheme: http
+      scrapeTimeout: 30s
+  namespaceSelector:
+    matchNames:
+      - external-secrets-operator
+  selector:
+    matchLabels:
+      app: external-secrets-operator
+      svc: external-secrets-operator-controller-manager-metrics-service
+----
+
+.. Create the `ServiceMonitor` CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f servicemonitor-external-secrets-operator-http.yaml
+----
++
+After the `ServiceMonitor` CR is created, the user workload Prometheus instance begins metrics collection from the Operator. The collected metrics are labeled with `job="external-secrets-operator-controller-manager-metrics-service"`.
+
+.Verification
+
+. In the {product-title} web console, navigate to *Observe* → *Targets*.
```

Comment:
I won't check any other instances, if you wouldn't mind checking the rest of the files in this PR

```suggestion
. In the {product-title} web console, navigate to *Observe* -> *Targets*.
```